build-scripts/for-linux/prepare-debian-based.bash

```diff
@@ -6,6 +6,8 @@ BASE_DIR=`dirname $0`
 
 sudo apt update
 
+sudo DEBIAN_FRONTEND=noninteractive apt-get install -y wget
```

Comment:
Why wget is necessary?

Reply:
need to download cmake 3.24 if debian 12 using for building